clojurebridge_attendee_care.md

```diff
@@ -0,0 +1,39 @@
+# Attendee care recipe
+
+`This is our recepe to attendee care. The tasks can be split. Different people can take over parts.`
+
+**Selection:**
+
+1. In our GoogleDrive:
+Leave the original table in its original state.
+Copy it completely and work in the copy.
+
+2. Select attendees by random. To do this generate random numbers from 1 to the amount of applicants. How many get invited depends on how many coaches signed up and is decided by all organizers and clarified before. Add an extra `status` field in the beginning of the table where you note who got `invited`.
+
+3. Send an invite mail to all selected ones. You can use [this template](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-1---to-the-selected) for the mail. You may have to adjust dates and some information. Set a deadline to reply the mail by 3 to 4 days.
+
+4. Send a [mail to the rest telling them they are on the waiting list](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-2---waitinglist).
+
+5. Check the mails frequently. Whenever you hear back from the invited people update their status to either `accepted` or `declined`.
+
+6. If a person declines [invite another random person who wasn't selected](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#inviting-from-waiting-list). Also for them set a deadline for 3 - 4 days.
+
+7. If somebody doesn't get back to any deadline you can send them [a mail](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#to-applicants-who-didnt-get-back) that we gave their spot to the next person.
+
+**Shortly before the event:**
+
+8. When you gathered all the information who attends start to match the groups. It's important to match for language and experience. *We do have attendees who can't speak English. Please consider this carefully.* Also it makes sense to match people who don't want to be photographed.
+
+9. 5 Days before the event: Send another [mail with details about the event](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-3---information-before-the-event).
+
+10. Before the event: prepare name tags with name and photo preference badge.
```

Comment:
Actually this was something I wanted to give feedback on and forgot. I think it's kind of nice for attendees to be able to customise their own name tag. I see the advantage of having someone prepared for everyone - especially with the photo preference sticker - but I think letting people pick what name they want their name tag to have is better. it's just a small thing but it's easy for us to do. In fact it is possibly less work for us this way! What do you think?